common/snapshots.py

```diff
@@ -677,6 +677,7 @@ def backup(self, force = False):
                                     gettext.ngettext('Waiting %s second.', 'Waiting %s seconds.', 30) % 30,
                                     30)
                         for counter in range(30, 0, -1):
+                            logger.info("Cannot start snapshot yet: target directory not accessible. Waiting 1s.")
```

Comment:
I'd just like to point out that since backintime by default logs at INFO this is potentially 30 non-informative log lines in syslog. A more sensible approach would be to either 1. not log this or 2. if logging this is crucial (I don't believe it is since the failure that can happen is logged below) then count how many times the iteration has happened and only log if it is greater than 1.